azure-pipelines.yml

```diff
@@ -430,6 +430,7 @@ jobs:
           for f in *; do
               gpg --homedir $GPG_DIR -ab $f
           done
+          sha256sum $(find . -type f | sort) > sha256sums
```

Comment:
How reasonable do you think it would be to manually add this for the last stable releases and snapshots since 1.2 (since with your and my pruning strategy we do not care about older snapshots after Wednesday)? So 1.0.0, 1.0.1 and 1.1.1?

Reply:
Very reasonable. I'll try to take care of it today.

Reply:
As discussed offline, moved the sha computation above the loop so the resulting file gets signed for future releases (obviously won't be for the manually added ones on older releases), as the sha of the signature files seems less valuable than the signature on the sha file.